packages/yoastseo/src/scoring/assessments/SCORING INCLUSIVE LANGUAGE.md

```diff
@@ -0,0 +1,28 @@
+# Overview of the inclusive language analysis scoring criteria
+
+For every (potentially) non-inclusive word or phrase that is found in the text, a red or orange bullet is shown.
+A red bullet means that the targeted word or phrase is never inclusive, regardless of context.
+An orange bullet means that the targeted word of phrase is _potentially_ non-inclusive, depending on the context in which it is used.
+
+The targeted words and phrases are divided into the following categories:
+* [age](inclusiveLanguage/configuration/ageAssessments.js)
+* [appearance](inclusiveLanguage/configuration/appearanceAssessments.js)
+* [culture](inclusiveLanguage/configuration/cultureAssessments.js)
+* [disability](inclusiveLanguage/configuration/disabilityAssessments.js)
+* [gender](inclusiveLanguage/configuration/genderAssessments.js)
+* [socio-economic status](inclusiveLanguage/configuration/sesAssessments.js)
+* [other](inclusiveLanguage/configuration/otherAssessments.js)
+
+The list of targeted words and phrases, together with the feedback strings that are shown for each found word/phrase,
+can be found in the files linked in the list above.
+
```

Comment:
For now does works. What do you think of hosting the database created with [the script of Martijn](https://github.com/Yoast/wordpress-seo/pull/19081) and linking to that? Might be more readable.